src/main/java/net/snowflake/ingest/streaming/OpenChannelRequest.java

```diff
@@ -40,6 +40,9 @@ public enum OnErrorOption {
   // Default timezone for TIMESTAMP_LTZ and TIMESTAMP_TZ columns
   private final ZoneId defaultTimezone;
 
+  private final String offsetToken;
+  private final boolean isOffsetTokenProvided;
```

Comment:
Could you explain why isOffsetTokenProvided is required?

Reply:
I think this is to distinguish null as an offset token vs. it not being set.

Reply:
Exactly, i want to support setting the offsetToken as null as well.

 In order to distinguish between default null string vs user provided null, I am using this extra flag.

Reply:
I see, could you add a IT test for NULL?